test/config.test.ts

```diff
@@ -58,6 +58,42 @@ test("Config Simple Override", () => {
   });
 });
 
+test("Config Override Stage By Id", () => {
+  const sampleConfig = {
+    environments: {
+      dev: {
+        resources: {
+          "Process Function": {
```

Comment:
Could the user specify `Stage/Process Function`? In case they had two instances of `SqsToLambdaStage`, but they only needed to configure one of them.

Reply:
Yep that will work, I can add a test for it as well.

Reply:
Added [test](https://github.com/awslabs/aws-ddk/pull/286/commits/dd905035f8acc1e8ee88f09ba4013093de2fb014) for this behavior.